test/dynamo/test_dynamo.py

```diff
@@ -31,16 +31,16 @@ def test_simple_model(self):
     res_cpu = self.fn_simple(x, y)
     res_xla_dynamo = self.fn_simple_dynamo(xla_x, xla_y)
     self.assertIn('xla::add', met.counter_names())
-    torch.allclose(res_cpu, res_xla_dynamo.cpu())
+    assert torch.allclose(res_cpu, res_xla_dynamo.cpu())
```

Comment:
Will it be better to use `self.assert`?

Reply:
Let me do that in a follow up pr lol, don't want to rerun ci since I want to backport this to 2.0 ASAP.